source/API/CMakeLists.txt

```diff
@@ -194,21 +194,23 @@ endif()
 
 # Copy the clang resource directory.
 add_custom_command_target(
-    unused_var
+    copy-clang-resource-dir
     COMMAND "${CMAKE_COMMAND}" "-E" "copy_directory" "${CLANG_RESOURCE_PATH}" "${lib_dir}/lldb/clang/"
     OUTPUT "${lib_dir}/lldb/clang/"
     VERBATIM
     ALL
     DEPENDS ${clang_headers_target})
 
 add_custom_command_target(
-    unused_var
+    copy-swift-resource-dir
     COMMAND "${CMAKE_COMMAND}" "-E" "copy_directory" "${SWIFT_RESOURCE_PATH}" "${lib_dir}/lldb/swift/"
     OUTPUT "${lib_dir}/lldb/swift/"
     VERBATIM
     ALL
     DEPENDS ${SWIFT_RESOURCE_PATH})
 
+add_dependencies(liblldb copy-clang-resource-dir copy-swift-resource-dir)
```

Comment:
I assume that liblldb and LLDB.framework are mutually copied, i.e., the resource dir isn't copied twice when building for macOS now?

Reply:
It will be copied twice, but to different locations. Once to the framework location and once to `lib/lldb/swift`.